README.md

```diff
@@ -5,17 +5,7 @@ Airbus UTM Platform `stack` CLI
 
 ### macOS/Linux one-liner 
 ```
-curl -sSL https://github.com/altiscope/platform-stack-cli/releases/download/v1.19.0/stack_$(
-    bash -c '[[ $OSTYPE == darwin* ]] && echo darwin || echo linux'
-  )_amd64 -o stack && chmod a+x stack && sudo mv stack /usr/local/bin/
-```
-
-### macOS
-```
-curl -sSL https://github.com/altiscope/platform-stack-cli/releases/download/v1.19.0/stack_darwin_amd64 -o stack && chmod a+x stack && sudo mv stack /usr/local/bin/
-```
-
-### Linux
-```
-curl -sSL https://github.com/altiscope/platform-stack-cli/releases/download/v1.19.0/stack_linux_amd64 -o stack && chmod a+x stack && sudo mv stack /usr/local/bin/
```

Comment:
Cleaning this up since we almost never use is. Cleaning up helps to maintain only one thing.